### PR TITLE
Making blitz changes from 5/19 and adding user notes 

### PR DESCRIPTION
**Pull request by ltnerap**
First pull request guys! Woohoo!

Once Katharine accepts the invite to the repo and Teti makes her account, I'll add them as reviewers as well.

### CODE REVIEW

File: wikiContent/automodConfig.md [hunk 1]
```diff
@@ -181,6 +230,6 @@
  comment: |
      Your submission has been removed because it did not contain a matching flair tag in the title. Please **resubmit** your post with an appropriate tag.
 
-     These are the currently available tags:
-     **[Question] [PSA] [Review] [Swatch] [FOTD] [Haul] [Fluff] [Discussion] [Glamday] [Discovery] [Roundup] [Deals] [Meetup] [Journal]**
-     If you have questions, [send us a mailmail!](https://www.reddit.com/message/compose?to=%2Fr%2FAsianBeauty)
+     These are the currently available tags:  
+     **[Question] [PSA] [Review] [Swatch] [FOTD] [Haul] [Fluff] [Discussion] [Glamday] [Discovery] [Roundup] [Deals] [Meetup] [Journal]**  
+     If you have questions, [send us a maod mail!](https://www.reddit.com/message/compose?to=%2Fr%2FAsianBeauty)
```
Comment (ltnerap) [inline wikiContent/automodConfig.md]: "maod mail" = "mod mail"

File: wikiContent/automodConfig.md [hunk 2]
```diff
@@ -181,6 +230,6 @@
  comment: |
      Your submission has been removed because it did not contain a matching flair tag in the title. Please **resubmit** your post with an appropriate tag.
 
-     These are the currently available tags:
-     **[Question] [PSA] [Review] [Swatch] [FOTD] [Haul] [Fluff] [Discussion] [Glamday] [Discovery] [Roundup] [Deals] [Meetup] [Journal]**
-     If you have questions, [send us a mailmail!](https://www.reddit.com/message/compose?to=%2Fr%2FAsianBeauty)
+     These are the currently available tags:  
+     **[Question] [PSA] [Review] [Swatch] [FOTD] [Haul] [Fluff] [Discussion] [Glamday] [Discovery] [Roundup] [Deals] [Meetup] [Journal]**
```
Comment (ltnerap) [inline wikiContent/automodConfig.md]: Remove PSA, Haul, Fluff tags

File: wikiContent/automodConfig.md [hunk 3]
```diff
@@ -16,6 +16,55 @@
  #Reported Items
  reports: 5
  modmail: The above {{kind}} by /u/{{author}} has received 5 reports. Please investigate.
+
+---
+ #Post Tag Removal
+ title (includes): ["haul", "hauls", "fluff", "psa","help", "PSA", "FLUFF", "HAUL", "HAULS"]
```
Comment (ltnerap) [inline wikiContent/automodConfig.md]: remove haul, hauls, fluff, psa, PSA, FLUFF, HAUL, HAULS

File: sidebar.md [hunk 4]
```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
...
-[DEALS](https://www.reddit.com/r/AsianBeauty/search?q=+flair:deals&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [MEETUPS](https://www.reddit.com/r/AsianBeauty/search?q=+meetups:meetups&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [JOURNAL](https://www.reddit.com/r/AsianBeauty/search?q=+flair:journal&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [SHOW ALL](http://goo.gl/9jxY49)
 
 ##Filter-Out Posts
 [QUESTION](http://qu.reddit.com/r/AsianBeauty/#qu)
-[SWATCHES & FOTD](http://sw.reddit.com/r/AsianBeauty/#sw)
+[SWATCHES](http://sw.reddit.com/r/AsianBeauty/#sw)
 [HAULS](http://ha.reddit.com/r/AsianBeauty/#ha)
```
Comment (ltnerap) [inline sidebar.md:67]: Do we want to remove hauls and fluff from this?

File: wikiContent/automodConfig.md [hunk 5]
```diff
@@ -7,7 +7,7 @@
  is_edited: false
  parent_submission:
      id: [2g3sqq, 34sij6]
- action: remove
+ action: remove
```
Comment (ltnerap) [inline wikiContent/automodConfig.md]: Can we remove this added space? I think it got put in mistakenly.

File: sidebar.md (hunk 4 above)
Comment (kstoops2conquer) [reply sidebar.md:67]: Because we're corralling hauls and fluff into specific days? (Look, I'm poking around and learning!)

File: sidebar.md [hunk 6]
```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 ##Filter Posts
 [MOD POST](https://www.reddit.com/r/AsianBeauty/search?q=+flair:mod-post&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [REVIEW](https://www.reddit.com/r/AsianBeauty/search?q=+flair:review&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[PSA](https://www.reddit.com/r/AsianBeauty/search?q=+flair:psa&restrict_sr=on&sort=new&t=all&feature=legacy_search)
```
Comment (kstoops2conquer) [inline sidebar.md:55]: Are we deprecating PSA entirely or will the modteam still be using it for selected mod posts? (I vote deprecate entirely)

File: subredditStylesheet.css [hunk 7]
```diff
@@ -5,10 +5,10 @@
 .author[href$="/AutoModerator"]:after{color: #9b59b6 !important; content: " Oshawott's Bot"}
 .author[href$="/biibiibeauty"]:after{color: #2c3e50 !important; content: " Wishtrend - Social Media Design"}
 .author[href$="/mishibox"]:after{color: #2c3e50 !important; content: " MISHIBOX Representative"}
-.author[href$="/ellumina"]:after{color: #9b59b6 !important; content: " Mod"}
+.author[href$="/ellumina"]:after{color: #9b59b6 !important; content: " Mod [currently on hiatus]"}
 .author[href$="/ashlaboo"]:after{color: #9b59b6 !important; content: " a wild modboo (•ㅅ•❀)"}
-.author[href$="/Ronrinesu"]:after{color: #9b59b6 !important; content: " Tomato Cheeks Flair Fairy"}
-.author[href$="/kstoops2conquer"]:after{color: #9b59b6 !important; content: " Friendly Grump"}
+.author[href$="/Ronrinesu"]:after{color: #9b59b6 !important; content: " Bird person / Bio nerd"}
+.author[href$="/kstoops2conquer"]:after{color: #9b59b6 !important; content: " Friendly Grump [currently on hiatus]"}
```
Comment (kstoops2conquer) [inline subredditStylesheet.css:11]: Shouldn't this say, "Friendly Grump / TrollMod?"

Kidding.

File: wikiContent/userNotes.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+/u/fairest-one-of-all
+[Spam Watch]new account; questionable results https://www.reddit.com/r/AsianBeauty/comments/66z213/fotd_acne_scars_after_2_months_of_15_step_korean/ - by /u/petitoignon - 25 days ago
+/u/DubuTutorials
+[Spam Watch]soliciting free business advice - by /u/petitoignon - 11 days ago
+/u/ModestNewbie
+Problematically reposted a bloggers non AB related content after the initial post was removed. - by /u/petitoignon - 10 days ago
+/u/Nina_in_ABwonderland
+[Spam Watch]tracking - by /u/petitoignon - a day ago
+/u/just_say_knope
+[Spam Watch]Possible Massdrop shill - by /u/Ronrinesu - about 11 hours ago
```
Comment (kstoops2conquer) [inline wikiContent/userNotes.md]: This whole section @ltnerap - none of the notes are rude, but they should probably remain private.

**Review by kstoops2conquer — APPROVED** [review]
Did I do it good?  (I liked your tutorial's explanation about the difference between "comment" and "approve.")

File: sidebar.md (hunk 4 above)
Comment (ltnerap) [reply sidebar.md:67]: Yeah, this is the thing that filters out posts with hauls or fluff flair. Which doesn't really exist anymore...

File: subredditStylesheet.css (hunk 7 above)
Comment (ltnerap) [reply subredditStylesheet.css:11]: Cackling.

File: wikiContent/userNotes.md (hunk 8 above)
Comment (ltnerap) [reply wikiContent/userNotes.md]: Agreed. File removed.

File: sidebar.md (hunk 6 above)
Comment (ltnerap) [reply sidebar.md:55]: I'm 99.9999999% sure we're deprecating it entirely

File: wikiContent/automodConfig.md (hunk 5 above)
Comment (ltnerap) [reply wikiContent/automodConfig.md]: Removed

File: wikiContent/automodConfig.md (hunk 2 above)
Comment (ltnerap) [reply wikiContent/automodConfig.md]: Removed

File: wikiContent/automodConfig.md (hunk 1 above)
Comment (ltnerap) [reply wikiContent/automodConfig.md]: Fixed

**Review by ltnerap — COMMENTED** [review]
Finished. I'm not allowed to approve the pull request since I was the one that created it, so I'm just commenting.

File: sidebar.md (hunk 6 above)
Comment (petitoignon) [reply sidebar.md:55]: I think we should deprecate entirely.

File: sidebar.md [hunk 9]
```diff
@@ -52,24 +52,18 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 ##Filter Posts
 [MOD POST](https://www.reddit.com/r/AsianBeauty/search?q=+flair:mod-post&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [REVIEW](https://www.reddit.com/r/AsianBeauty/search?q=+flair:review&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[PSA](https://www.reddit.com/r/AsianBeauty/search?q=+flair:psa&restrict_sr=on&sort=new&t=all&feature=legacy_search)
 [QUESTION](https://www.reddit.com/r/AsianBeauty/search?q=+flair:question&restrict_sr=on&sort=new&t=all&feature=legacy_search)
-[SWATCHES & FOTD](https://www.reddit.com/r/AsianBeauty/search?q=+flair:swatches-fotd&restrict_sr=on&sort=new&t=all&feature=legacy_search)
```
Comment (petitoignon) [inline sidebar.md:57]: I think we should revisit FOTD only but with a few more rules around how to post ie general routine and what the photo is a of/ a blurb

File: sidebar.md [hunk 10]
```diff
@@ -87,19 +81,38 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
 
 
 ##Posting Schedule
-**Daily** - Routine Help & Troubleshooting
+Each post starts at 00:00 EST
```
Comment (petitoignon) [inline sidebar.md:84]: I have a dumb question...how do we set 00:00 EST?

File: sidebar.md [hunk 11]
```diff
@@ -87,19 +81,38 @@ We actively recruit people from diverse time zones and backgrounds, who have goo
...
+**Monday** - Weekly Deals
+
 **Tuesday** - Discovery Tuesday: New finds, discoveries
 
-**Wednesday** - Hump-Day: Hauls & Destashing
+**Tuesday** - Unique Discussion (new topic every week)
+
+**Wednesday** - Hump-Day Hauls
+
+**Wednesday** - Wednesday Help Thread
```
Comment (petitoignon) [inline sidebar.md:96]: Does this need to be removed?

**Comment by ltnerap** [comment]
Merging this pull request as is. Note that this is out of date and this issue will be fixed by subsequent pull requests.

**Pull request merged**